frontend/src/metabase/core/components/Button/Button.tsx

```diff
@@ -127,7 +128,7 @@ const BaseButton = forwardRef(function BaseButton(
   );
 });
 
-const Button = styled(BaseButton)`
+const Button = styled(BaseButton)<SpaceProps>`
```

Comment:
After `<ConfirmContent />` has been converted to TypeScript it starts checking for type within that file that we use `ml` on Button so I need to add its props here.
https://github.com/metabase/metabase/blob/40bfc4a5543ba3f0fa1a23b97a4b0f6b5941ace9/frontend/src/metabase/components/ConfirmContent.tsx#L57